sympy/physics/continuum_mechanics/truss.py

```diff
@@ -0,0 +1,620 @@
+
+        new_label: String or Symbol
+            The new label of the member.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.truss import Truss
+        >>> t = Truss()
+        >>> t.add_node('A', 0, 0)
+        >>> t.add_node('B', 3, 0)
+        >>> t.nodes
+        [('A', 0, 0), ('B', 3, 0)]
+        >>> t.change_node_label('A', 'C')
+        >>> t.nodes
+        [('C', 0, 0), ('B', 3, 0)]
+        >>> t.add_member('BC', 'B', 'C')
+        >>> t.members
+        {'BC': ['B', 'C']}
+        >>> t.change_member_label('BC', 'BC_new')
+        >>> t.members
+        {'BC_new': ['B', 'C']}
+        """
+        if label not in list(self._members):
+            raise ValueError("No such member exists for the Truss")
+
+        else:
+            members_duplicate = list(self._members).copy()
+            for member in members_duplicate:
+                if member == label:
+                    self._members[new_label] = [self._members[member][0], self._members[member][1]]
+                    self._members.pop(label)
+                    self._internal_forces[new_label] = self._internal_forces[label]
+                    self._internal_forces.pop(label)
+
+    def apply_load(self, location, magnitude, direction):
+        """
+        This method applies an external load at a particular node
+
+        Parameters
+        ==========
+        location: String or Symbol
+            Label of the Node at which load is applied.
+
+        magnitude: Sympifyable
+            Magnitude of the load applied. It must always be positive and any changes in
+            the direction of the load are not reflected here.
+
+        direction: Sympifyable
+            The angle, in degrees, that the load vector makes with the horizontal
+            in the counter-clockwise direction. It takes the values 0 to 360,
+            inclusive.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.truss import Truss
+        >>> from sympy import symbols
+        >>> t = Truss()
+        >>> t.add_node('A', 0, 0)
+        >>> t.add_node('B', 3, 0)
+        >>> P = symbols('P')
+        >>> t.apply_load('A', P, 90)
+        >>> t.apply_load('A', P/2, 45)
+        >>> t.apply_load('A', P/4, 90)
+        >>> t.loads
+        {'A': [[5*P/4, 90], [P/2, 45]]}
+        """
+        magnitude = sympify(magnitude)
+        direction = sympify(direction)
+
+        if location not in self.node_labels:
+            raise ValueError("Load must be applied at a known node")
+
+        else:
+            if location in list(self._loads):
+                if direction in [self._loads[location][i][1] for i in range(len(self._loads[location]))]:
+                    for i in range(len(self._loads[location])):
+                        if self._loads[location][i][1] == direction:
+                            self._loads[location][i][0] += magnitude
+                            break
```

Comment:
I don't think you should add loads that act in the same direction. It would be better to simply append them.
Right now the following code throws an error
```
>>> from sympy.physics.continuum_mechanics.truss import Truss
>>> t = Truss()
>>> t.add_node("node_1", 0, 0)
>>> t.apply_load("node_1", magnitude=10, direction=0)
>>> t.apply_load("node_1", magnitude=10, direction=0)
>>> t.remove_load("node_1", magnitude=10, direction=0)